code/game/objects/items/weapons/storage/deferred.dm

```diff
@@ -90,6 +90,12 @@
 	icon_state = "box_serbian"
 	initial_contents = list(/obj/item/device/synthesized_instrument/trumpet = 1) //TODO: Add an accordian to this, sprites already made.
 
+/obj/item/weapon/storage/deferred/disks //for the beacon, cause we want it priced right and actually functional
+	name = "autolathe disk box"
+	desc = "A small collection of autolathe disks"
+	initial_contents = list(/obj/spawner/lathe_disk = 7)
+	price_tag = 2800
```

Comment:
I suggest making it cost way way more than 2800

Reply:
On the beacon, it’s about 3.3 thousand. I think that’s enough

Reply:
I'm trying to wrack my brain for the most expensive disk I've ever bought from someone. I think the max I usually pay is 1-2k. So 3.3 sounds fair.

Reply:
This is 7 disks with some of them having the possibility of being an extremely rare one that is blacklisted from spawning in loot. The fact alone that this "lootbox" costs so little when it has the ability to print out an AMR, a Dallas or some other godforsaken gun we haven't thought about is the issue, not how much you pay for disks. You can print 5-10 guns out from a disk, that's worth more than just the disk on its own when the guild has access to minerals.

Reply:
Maint loot also has possibility of Dallas and other similar disks, moot point, unless the gotten disks are infinite(excelsior ones) the 5-10 number is absolutely false and  is 4 for most disks, regarding access to minerals Moebius has a barge that can go to the asteroid too, not a problem, the current price is absolutely alright especially considering the 200 credit shipping fee.

Reply:
> This is 7 disks with some of them having the possibility of being an extremely rare one that is blacklisted from spawning in loot. The fact alone that this "lootbox" costs so little when it has the ability to print out an AMR, a Dallas or some other godforsaken gun we haven't thought about is the issue, not how much you pay for disks. You can print 5-10 guns out from a disk, that's worth more than just the disk on its own when the guild has access to minerals.

Only /obj/spawner/lathe_disk/advanced can spawn  those , basic ones cannot.

Reply:
Also let's not forget the lathe nerfs. A new disk can print 4 guns, but those gun will come out funky. And if it's a gun with a unique mag you are going to have to waste a ton of license points since mags no longer print fully loaded.

Reply:
>  especially considering the 200 credit shipping fee.

The shipping fee is the price of the crate, it's just there so you don't order and export unlimited crates.